src/objects/OrientationGuide.ts

```diff
@@ -0,0 +1,144 @@
+import { Scene } from "phaser";
+import * as CONFIG from "../utils/configuration";
+
+export class OrientationGuide {
+    private scene: Scene;
+    private overlay: Phaser.GameObjects.Container;
+    private pulseEffect: Phaser.Tweens.Tween | null = null;
+    
+    // Static flag to track if the guide has been shown during this game session
+    private static hasBeenShownBefore: boolean = false;
+
+    constructor(scene: Scene) {
+        this.scene = scene;
+        
+        // Only check orientation in the MainMenu scene and only if we haven't shown it before
+        if (scene.scene.key === 'MainMenu' && !OrientationGuide.hasBeenShownBefore) {
```

Comment:
you probably don't need to check the scene key, you are only adding it to main menu.